src/components/apiExamples/apiExamples.tsx

```diff
@@ -25,29 +29,39 @@ function Example({api, selectedTabView, selectedResponse}: ExampleProps) {
     exampleJson = api.responses[selectedResponse].content?.example;
   }
 
+  // load prism dynamically for these codeblocks,
+  // otherwise the highlighting applies globally
+  useEffect(() => {
```

Comment:
Will this cause layout shift or flashing because it paints some time after the initial page load? 🤔

Reply:
Yes, but it did that before as well. We could also think about not dynamically loading prismjs at all and instead use what we already have (rehypePrismPlus). This would streamline the style of the API JSON examples with our other codeblocks. This would require more work though, we can do it in another pr if needed.